packages/@okta/vuepress-theme-default/global-components/EventTypes.vue

```diff
@@ -78,13 +90,31 @@
         return value.replace(/[\s_.]/g, '');
       }
     },
-
     watch: {
-      search: _.debounce((value) => {
+      search() {
+        this.addHistory()
+      },
+      release() {
+        this.addHistory()
+      }      
+    },
+    methods: {
+      updateSearch: _.debounce(function(e) {
+        this.search = e.target.value
+      }, 100),
+      addHistory() {
         if (history.pushState) {
-          history.pushState(null, '', '?q=' + encodeURI(value));
+          if (this.search && this.release) {
+            history.pushState(null, '', '?q=' + encodeURI(this.search) + '&release=' + encodeURI(this.release))
+          } else if (this.search) {
+            history.pushState(null, '', '?q=' + encodeURI(this.search))
+          } else if (this.release) {
+            history.pushState(null, '', '?release=' + encodeURI(this.release))
+          } else {
+            history.pushState(null, '', this.$route.path)
+          }
```

Comment:
I tried:

```javascript
        this.$router.push({
          path: '',
          query: { 
            q: this.search || undefined,
            release: this.release || undefined
          }
        })
```

which works, but bounces to the top of viewport :/ Seem to be a known issue.